stellargraph/connector/neo4j/graph.py

```diff
@@ -0,0 +1,97 @@
+__all__ = ["Neo4jStellarGraph"]
```

Comment:
very boring I'm sorry, but this needs the copyright header
```suggestion
# -*- coding: utf-8 -*-
#
# Copyright 2018-2020 Data61, CSIRO
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#   http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

__all__ = ["Neo4jStellarGraph"]
```

Reply:
yep i always forget the copyright header!